### DISCUSSION TITLE
Provide demo videos

### DISCUSSION THREAD

**Discussion by juliapissarro** [Dashlane/dashlane-cli#140] [Ideas]
In addition to the already available CLI documentation, it would be great if we could have a few short demos to show:

- How to use the CLI from an end user's standpoint? e.g. copy a password, copy an SSH key
- How to use the CLI from an admin's standpoint? e.g. generate a report of logs, filter the logs, export the logs into a SIEM

**Comment by Mikescops** [comment]
We started a series of videos on Youtube explain how to use the CLI: https://www.youtube.com/playlist?list=PLaJHdXnJiAn4rWs-YlmKuoX__33BAq_xf

I also recommend having a look at our documentation website where some use cases are detailed: https://dashlane.github.io/dashlane-cli
